### ISSUE TITLE
Footer Sitemap does not work

### ISSUE THREAD

**Issue by expressionstech** [tonystar/drupal-tweme#40]
I created a new menu, named it Footer Sitemap. I edited the machine name to footer_sitemap however i get an error message "The machine-readable name must contain only lowercase letters, numbers, and hyphens.". It will not allow the _ character, only a hyphen. When i set the machine name to footer-sitemap, nothing is generated in the footer. Am I doing something wrong or missing something?

**Comment by tonystar** [comment]
No, you are doing right way. Have you followed the step 2: Place related menu block in the Footer region?

What about `footer_sitemap` — this was my mistake, of course it should be `footer-sitemap`. Updated the documentation.

**Comment by expressionstech** [comment]
Yes, I have added the menu to the footer block.

On Tue, Oct 6, 2015 at 5:24 PM, Anton Staroverov <notifications@github.com>
wrote:

> No, you are doing right way. Have you followed the step 2: Place related
> menu block in the Footer region?
>
> What about footer_sitemap — this was my mistake, of course it should be
> footer-sitemap. Updated the documentation.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tonystar/drupal-tweme/issues/40#issuecomment-146006462>
> .
>



-- 

*Jeff Duke* - President
Expressions Technologies, LLC
*Expressions Tributes | Gifts of Sympathy | Expressions Flowers*
Office - 877.410.6513 *|* Fax - 724.798.0430 *| Email - *
duke@expressionstech.com

THE INFORMATION CONTAINED IN THIS E-MAIL MESSAGE, TOGETHER WITH ANY
ATTACHMENT HERETO, IS PRIVILEGED AND CONFIDENTIAL AND IS INTENDED ONLY FOR
THE USE OF THE INDIVIDUAL(S) NAMED ABOVE AND OTHERS WHO HAVE BEEN
SPECIFICALLY AUTHORIZED TO RECEIVE SUCH E-MAIL. IF THE RECIPIENT IS NOT THE
INTENDED RECIPIENT, YOU ARE HEREBY NOTIFIED THAT ANY DISSEMINATION,
DISTRIBUTION OR COPY OF THIS COMMUNICATION IS STRICTLY PROHIBITED. IF YOU
HAVE RECEIVED THIS COMMUNICATION IN ERROR, PLEASE IMMEDIATELY NOTIFY THE
SENDER BY TELEPHONE AT THE NUMBER NOTED ABOVE.

**Comment by tonystar** [comment]
Did you create any menu items?
Is your website available online? Could you give me a temporary access?
Otherwise — could you make a screenshot of 1) menu links 2) block configuration.

**Comment by qupro** [comment]
It doesn't work for me either. 
Maybe it could be related to the fact that drupal [adds a menu- prefix by default](https://api.drupal.org/api/drupal/modules%21menu%21menu.module/constant/MENU_MAX_MENU_NAME_LENGTH_UI/7) so machine name results always "menu-footer-sitemap".

**Comment by mediacourses** [comment]
The same issue here we cant seem to get the site footer site map working. Could you please provide step by step as the documentation is not very clear with screen grabs? Thanks.
